Gemfile

```diff
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 gem 'fastlane'
 gem 'snapshot'
 gem 'cocoapods', '~> 1.10'
+gem 'jazzy'
```

Comment:
Consider specifying a version range so we don’t find out about a new major version when it breaks and interrupts a release process.

Reply:
Good catch, pinned it to `~> 0.14`.